### PR TITLE
chore: clarify contributing guidelines

### PR DESCRIPTION
**Pull request by sjswerdlow**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@sjswerdlow
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: README.md [hunk 1]
```diff
@@ -53,11 +53,6 @@ resources of this module:
 The [Project Factory module][project-factory-module] can be used to
 provision a project with the necessary APIs enabled.
 
-## Contributing
```
Comment (bharathkkb) [inline README.md:56]: Could we clarify we are not accepting in the readme and also delete https://github.com/terraform-google-modules/terraform-google-sap/blob/master/CONTRIBUTING.md?

**Comment by comment-bot-dev** [comment]
@sjswerdlow
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@sjswerdlow
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**